### PR TITLE
simulation: add note how to run SITL faster

### PR DESCRIPTION
**Pull request by julianoes**
These adds docs on how to run the SITL simulation at a faster than realtime rate.

### CODE REVIEW

File: en/simulation/README.md [hunk 1]
```diff
@@ -102,6 +102,26 @@ The simulation can be further configured via environment variables:
 The syntax described here is simplified, and there are many other options that you can configure via *make* - for example, to set that you wish to connect to an IDE or debugger. For more information see: 
 [Building the Code > PX4 Make Build Targets](../setup/building_px4.md#make_targets).
 
+### Run Simulation Faster than Realtime
+
+SITL can be run faster or slower than realtime. The speed factor is set using the environment variable `PX4_SIM_SPEED_FACTOR`.
+The speed factor is implemented for the jMAVSim and Gazebo SITL simulation.
+
+For instance to run the jMAVSim simulation at 2 times the real time speed, do:
```
Comment (hamishwillee) [inline en/simulation/README.md]: For instance, to run the jMAVSim simulation at 2 times the real time speed:

**Comment by hamishwillee** [comment]
Looks great. Minor rendering issues I might play with.

Some points/questions
- "The speed factor is implemented for the jMAVSim and Gazebo SITL simulation." 
  - So this is something that requires both a PX4 sitl change and a simulator change?
  - Perhaps we should link to this section from http://dev.px4.io/en/simulation/jmavsim.html#usageconfiguration-options and http://dev.px4.io/en/simulation/gazebo.html#usageconfiguration-options 
- When would you want to do this? I presume it allows you to speed up automated tests? Anything else?
- Are there any limitations to do with using this - ie say you have a developer API implementing a protocol that has timeouts. If you run simulator at 10x speed might the timeouts expire too early. What if someone is using a joystick through QGC, will that result in jittery behaviour. These cases might not be realistic - I'm basically trying to get you to think about possible fail cases.

**Comment by julianoes** [comment]
- The speed factor is obeyed by jMAVSim and Gazebo, so they are the ones reading the environment variable. You're right it should be in the configuration options, at least as well.
- I would always use it. When quickly testing something, it's pretty boring to watch the drone in real time.
- That's possible fail cases, yes.

**Comment by hamishwillee** [comment]
@julianoes Thanks. I merged then made some minor editorial changes in #715, mostly to link back to the more detailed changes in this PR.

Further things that would be good to see:
- Add an example of slowing the simulation - ie is that done using a fractional value, or a negative value?
- Add some info about when it should be used and the risk/problem symptoms. ie I know you said use it all the time and that is the advice, then we should give it. Otherwise why not just set the factor to be 10 and be done with it :-)

**Comment by julianoes** [comment]
- Slowing down to half would be 0.5.
- We'll find out :smile: .

**Comment by hamishwillee** [comment]
Indeed we will. I've added info on half time in https://github.com/PX4/Devguide/commit/04b6d4f2f61142b7c67ef51dfc46bde9f29e4763

**Comment by julianoes** [comment]
Great, thanks!

**Pull request merged**